### ISSUE TITLE
Is it normal to re-train the entire model (all parameters) for fine-tuning?

### ISSUE THREAD

**Issue by ldorigo** [facebookresearch/detectron2#4805]
## 📚 Documentation Issue

In the colab notebook linked from the readme (https://colab.research.google.com/drive/16jcaJoc6bCFAQ96jDe2HwtXj7BMD_-m5), there is a section about fine-tuning a model from the model zoo:
 
> "Now, let's fine-tune a COCO-pretrained R50-FPN Mask R-CNN model on the balloon dataset."

However it looks like the entire model is actually retrained - `sum(p.numel() for p in trainer.model.parameters() if p.requires_grad)` gives a number that is essentially the entirety of the model's parameters. This is not how I would normally understand fine tuning, where only the "last layers" (whatever those are in context) are re-trained? 

This may just be because I am not very familiar with detection/segmentation tasks, but is it normal to retrain the entire model for a fine-tuning/transfer learning task?
